src/shared/components/Input/component.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  InputRootContainer,
+  InputContainer,
+  InputWrapper,
+  InputValidationMessageWrapper,
+  InputGroupButton,
+} from './styles';
```

Comment:
please make them one-liner to save space, like
```js
import { InputRootContainer, InputContainer, InputWrapper, InputValidationMessageWrapper, InputGroupButton } 
  from './styles';

// or if it's not enough space

import { InputRootContainer, InputContainer, InputWrapper, InputValidationMessageWrapper, 
  InputGroupButton } from './styles';
```

Reply:
It's prettier settings.  When I try to change the format it's automatically changed to this.

Reply:
oh, I see, I think we should fix this in future.